pyproject.toml

```diff
@@ -47,6 +29,36 @@ Source = "https://github.com/IntelLabs/MART"
 mart = "mart.__main__:main"
 
 [project.optional-dependencies]
+
```

Comment:
I would add `full` to make installing everything easy:

```
full = [
  "mart[core,vision,objdet,loggers,developer]",
]
```

You can then use this whenever you run tests.

Reply:
Good idea. Added in 0f3e307d563cdbf14662e3aa07433858b5fb5b80

Reply:
I had to comment out the loggers dependency to avoid lengthy(more than 9 hours) resolution. They were already commented out previously in the required dependency.